clients/go/pkg/pb/gateway.pb.go

```diff
@@ -4716,52 +5077,58 @@ var file_gateway_proto_depIdxs = []int32{
-	44, // 37: gateway_protocol.Gateway.UpdateJobRetries:input_type -> gateway_protocol.UpdateJobRetriesRequest
-	48, // 38: gateway_protocol.Gateway.ModifyProcessInstance:input_type -> gateway_protocol.ModifyProcessInstanceRequest
-	50, // 39: gateway_protocol.Gateway.DeleteResource:input_type -> gateway_protocol.DeleteResourceRequest
-	52, // 40: gateway_protocol.Gateway.BroadcastSignal:input_type -> gateway_protocol.BroadcastSignalRequest
-	4,  // 41: gateway_protocol.Gateway.ActivateJobs:output_type -> gateway_protocol.ActivateJobsResponse
-	5,  // 42: gateway_protocol.Gateway.StreamActivatedJobs:output_type -> gateway_protocol.ActivatedJob
-	7,  // 43: gateway_protocol.Gateway.CancelProcessInstance:output_type -> gateway_protocol.CancelProcessInstanceResponse
-	9,  // 44: gateway_protocol.Gateway.CompleteJob:output_type -> gateway_protocol.CompleteJobResponse
-	12, // 45: gateway_protocol.Gateway.CreateProcessInstance:output_type -> gateway_protocol.CreateProcessInstanceResponse
-	14, // 46: gateway_protocol.Gateway.CreateProcessInstanceWithResult:output_type -> gateway_protocol.CreateProcessInstanceWithResultResponse
-	16, // 47: gateway_protocol.Gateway.EvaluateDecision:output_type -> gateway_protocol.EvaluateDecisionResponse
-	23, // 48: gateway_protocol.Gateway.DeployProcess:output_type -> gateway_protocol.DeployProcessResponse
-	26, // 49: gateway_protocol.Gateway.DeployResource:output_type -> gateway_protocol.DeployResourceResponse
-	33, // 50: gateway_protocol.Gateway.FailJob:output_type -> gateway_protocol.FailJobResponse
-	35, // 51: gateway_protocol.Gateway.ThrowError:output_type -> gateway_protocol.ThrowErrorResponse
-	37, // 52: gateway_protocol.Gateway.PublishMessage:output_type -> gateway_protocol.PublishMessageResponse
-	39, // 53: gateway_protocol.Gateway.ResolveIncident:output_type -> gateway_protocol.ResolveIncidentResponse
-	47, // 54: gateway_protocol.Gateway.SetVariables:output_type -> gateway_protocol.SetVariablesResponse
-	41, // 55: gateway_protocol.Gateway.Topology:output_type -> gateway_protocol.TopologyResponse
-	45, // 56: gateway_protocol.Gateway.UpdateJobRetries:output_type -> gateway_protocol.UpdateJobRetriesResponse
-	49, // 57: gateway_protocol.Gateway.ModifyProcessInstance:output_type -> gateway_protocol.ModifyProcessInstanceResponse
-	51, // 58: gateway_protocol.Gateway.DeleteResource:output_type -> gateway_protocol.DeleteResourceResponse
-	53, // 59: gateway_protocol.Gateway.BroadcastSignal:output_type -> gateway_protocol.BroadcastSignalResponse
-	41, // [41:60] is the sub-list for method output_type
-	22, // [22:41] is the sub-list for method input_type
-	22, // [22:22] is the sub-list for extension type_name
-	22, // [22:22] is the sub-list for extension extendee
-	0,  // [0:22] is the sub-list for field type_name
+	58, // 19: gateway_protocol.ModifyProcessInstanceRequest.activateInstructions:type_name -> gateway_protocol.ModifyProcessInstanceRequest.ActivateInstruction
```

Comment:
:D When was this the last time generated 😅

Reply:
I think before the 8.3 release as part of the MT epic, why do you ask?